ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -66,6 +160,47 @@ define(['QUnit', 'jquery', 'setup_manage_taxonomies', 'reactaddons',
             repoSlug="repo"
           />
         );
+      //test initial state
+      assert.equal(
+        vocabularyComponentRendered.getInitialState().terms.length,
+         2
+      );
+      // check term title
+      var listOfLinks = React.addons.TestUtils.
+        scryRenderedDOMComponentsWithTag(
+          vocabularyComponentRendered,
+          'a'
+      );
+      assert.equal(listOfLinks.length, 2);
+      var linkHeader = listOfLinks[0].getDOMNode();
+      assert.equal(linkHeader.innerHTML, vocabulary.name);
+      //test items
+      var devItems = React.addons.TestUtils.
+        findRenderedDOMComponentWithClass(
+          vocabularyComponentRendered,
+          'panel-body'
+      );
+      var itemList = React.addons.TestUtils.
+        scryRenderedDOMComponentsWithTag(
+          devItems,
+          'li'
+      );
+      assert.equal(itemList.length, 2);
+      //test enter text in input text
+      var inputNode = React.addons.TestUtils.
+        findRenderedDOMComponentWithTag(
+          vocabularyComponentRendered,
+          'input'
+      );
+      React.addons.TestUtils.Simulate.change(
+        inputNode,
+        {target: {value: 'test12'}}
+      );
+      assert.equal(
+        'test12',
+        vocabularyComponentRendered.state.newTermLabel
+      );
+      //test enter key
       var randomEvent = $.Event("keyup", {keyCode: 64});
       var eventEnter = $.Event("keyup", {keyCode: 13});
```

Comment:
It is recommended to use `React.addons.TestUtils.Simulate.keyUp` instead here since React will pass instances of `SyntheticEvent` to event handlers and not regular javascript events like you have here.